src/tardev-snapshotter/tarindex/src/lib.rs

```diff
@@ -136,18 +137,12 @@ fn read_all_entries(
                             .link_name_bytes()
                             .unwrap_or(std::borrow::Cow::Borrowed(b""));
                         if *hname != *name {
-                            // TODO: Handle this case by duplicating the full name.
-                            eprintln!(
-                                "Skipping symlink with long link name ({}, {} bytes, {}, {} bytes): {}",
-                                String::from_utf8_lossy(&name), name.len(),
-                                String::from_utf8_lossy(&hname), hname.len(),
-                                String::from_utf8_lossy(&f.path_bytes())
-                            );
-                            continue;
+                            special_link.push((&name).to_vec());
```

Comment:
Can't it just be `name.to_vec()`? That is, omitting parentheses and ampersand?

Reply:
Good catch! Yes. Code simplified.